core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/TransformCorrelatedGlobalAggregationWithProjection.java

```diff
@@ -200,7 +212,7 @@ public Result apply(CorrelatedJoinNode correlatedJoinNode, Captures captures, Co
                 ImmutableList.of(),
                 inputWithUniqueId.getOutputSymbols(),
                 source.getOutputSymbols(),
-                false,
```

Comment:
It should happen by itself with other rule. I think at this point it will get erased with reorder joins. Please validate it still works when you set it to `false`

Reply:
Without setting this flag it directly, it is not set for left join.

Reply:
> Without setting this flag it directly, it is not set for left join.

Could you check why? The rule is `OptimizeDuplicateInsensitiveJoins`